sigridci/sigridci.py

```diff
@@ -45,6 +46,32 @@ class UploadOptions:
     includeHistory: bool = False
     pathPrefix: str = ""
     showContents: bool = False
+    
+    
+@dataclasses.dataclass
+class TargetQuality:
+    ratings: typing.Dict[str, float]
+
+    def __init__(self, configFile, targetRating):
+        self.ratings = {"MAINTAINABILITY" : targetRating}
+        
+        if os.path.exists(configFile):
+            log(f"Loading target quality level from configuration file {configFile}")
+            # We can't use pyyaml because PIP is not available in the some of the
+            # very diverse set of customer environments where Sigrid CI is used.
+            targetPattern = re.compile("(" + "|".join(Report.METRICS) + "):\s*([\d\.]+)", re.IGNORECASE)
+            for line in open(configFile, "r"):
+                match = targetPattern.match(line.strip())
+                if match:
+                    self.ratings[match.group(1).upper()] = float(match.group(2))
+    
+    def isPassed(self, feedback, metric):
```

Comment:
Would it be an idea to rename this method to something more descriptive like: `meetsTargetQualityForMetric`?

Reply:
You're right, will rename this one and the other method name you suggested.